apps/src/music/progress/ConditionsChecker.ts

```diff
@@ -41,6 +54,11 @@ export default class ConditionsChecker {
   }
 }
 
-export interface KnownConditions {
+export interface KnownCondition {
```

Comment:
nit: mind moving these to the top of the file?

Reply:
Will do.